[2024-03-02T16:52:09.830+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T00:10:00+00:00 [queued]>
[2024-03-02T16:52:09.842+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T00:10:00+00:00 [queued]>
[2024-03-02T16:52:09.843+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-03-02T16:52:09.844+0000] {taskinstance.py:1363} INFO - Starting attempt 9 of 12
[2024-03-02T16:52:09.844+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-03-02T16:52:09.861+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): Scrapping> on 2024-03-02 00:10:00+00:00
[2024-03-02T16:52:09.872+0000] {standard_task_runner.py:55} INFO - Started process 589 to run task
[2024-03-02T16:52:09.875+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Batching_Data', 'Scrapping', 'scheduled__2024-03-02T00:10:00+00:00', '--job-id', '276', '--raw', '--subdir', 'DAGS_FOLDER/Batching_Data.py', '--cfg-path', '/tmp/tmpyn84adoa']
[2024-03-02T16:52:09.877+0000] {standard_task_runner.py:83} INFO - Job 276: Subtask Scrapping
[2024-03-02T16:52:09.965+0000] {task_command.py:376} INFO - Running <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T00:10:00+00:00 [running]> on host 76a23c179f72
[2024-03-02T16:52:10.082+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bob Sebastian
AIRFLOW_CTX_DAG_ID=Batching_Data
AIRFLOW_CTX_TASK_ID=Scrapping
AIRFLOW_CTX_EXECUTION_DATE=2024-03-02T00:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=9
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-02T00:10:00+00:00
[2024-03-02T16:52:18.455+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/base.py:338: UserWarning: Trying to unpickle estimator LabelEncoder from version 1.2.2 when using version 1.0.2. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:
https://scikit-learn.org/stable/modules/model_persistence.html#security-maintainability-limitations
  UserWarning,

[2024-03-02T16:52:18.965+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Batching_Data.py", line 93, in batching_data
    # Create a file-like object from the image data
NameError: name 'io' is not defined
[2024-03-02T16:52:18.981+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=Batching_Data, task_id=Scrapping, execution_date=20240302T001000, start_date=20240302T165209, end_date=20240302T165218
[2024-03-02T16:52:18.997+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 276 for task Scrapping (name 'io' is not defined; 589)
[2024-03-02T16:52:19.059+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2024-03-02T16:52:19.089+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
